service/matching/db.go

```diff
@@ -46,26 +46,33 @@ import (
 const (
 	initialRangeID     = 1 // Id of the first range of a new task queue
 	stickyTaskQueueTTL = 24 * time.Hour
+
+	userDataEnabled userDataEnabledState = iota
+	userDataDisabled
+	userDataSpecificVersion
```

Comment:
I think we should disambiguate these. User data could be:
- Enabled
- Disabled for this tqm because it manages a specific version set (so it will never change)
- Disabled due to a setting on this node or a response it received from a parent (so it could change in the future)

Reply:
that's what those are. I'll add comments